invenio_theme/static/scss/invenio_theme/styles.scss

```diff
@@ -0,0 +1,13 @@
+@import "variables.scss";
```

Comment:
This can be ' @import "variable" '. The extension is generally left out in scss. Same for the imports below.